sdk/src/client/node_api/core/routes.rs

```diff
@@ -330,12 +330,19 @@ impl Client {
 
     /// Finds an output, as JSON, by its OutputId (TransactionId + output_index).
     /// GET /api/core/v2/outputs/{outputId}
-    pub async fn get_output(&self, output_id: &OutputId) -> Result<OutputWithMetadataResponse> {
+    pub async fn get_output(&self, output_id: &OutputId) -> Result<OutputWithMetadata> {
         let path = &format!("api/core/v2/outputs/{output_id}");
 
-        self.node_manager
+        let response: OutputWithMetadataResponse = self
+            .node_manager
             .get_request(path, None, self.get_timeout(), false, true)
-            .await
+            .await?;
+
+        // ? can this be unverified?
```

Comment:
Yes, should use the verified version, but is already async, so no change needed there? Because you wrote
> But probably we should make additional async and get token_supply

Reply:
fixed, thanks